pytorch_lightning/trainer/optimizers.py

```diff
@@ -64,6 +64,9 @@ def init_optimizers(self, model: LightningModule) -> Tuple[List, List, List]:
         elif isinstance(optim_conf, (list, tuple)):
             optimizers = list(optim_conf)
         # unknown configuration
+        elif PYSYFT_AVAILABLE:
```

Comment:
can't we just extend the condition of the if statement in the top? Since we're doing the same if the condition holds.